web/src/app/registration/registration.component.scss

```diff
@@ -7,6 +7,42 @@ form, #formContainer {
     align-items: center;
     font-family: $defaultFont;
```

Comment:
You can move definition of `font-family` to `styles.scss` and add it to `body` element there, so that the default font is defined across the whole app instead of page-by-page basis.